### DISCUSSION TITLE
Filtering get_all

### DISCUSSION THREAD

**Discussion by drc0** [awtkns/fastapi-crudrouter#179] [Q&A]
Hello, 
I was wondering what was the better way to implement a custom filtering on get_all route, for example filtering by date, or related object property (relationship on sqlalchemy object). Thanks!

**Comment by afparsons** [comment]
It is my reading that filtering is not currently implemented; [see this issue](https://github.com/awtkns/fastapi-crudrouter/issues/62).

Have you been able to achieve filtering? If so, I am curious to know how.

**Reply by iagobalmeida** [reply]
Hey there, i've just sent my solution on the #62 topic!

Hope it helps you!
